src/pages/registrationScreen/RegistrationScreen.jsx

```diff
@@ -84,6 +90,7 @@ export function RegistrationScreen() {
       console.log(error);
       alert("Não foi possível registrar seu produto :(");
     }
+    goToHome();
```

Comment:
If error happened, is it go to home too ? Maybe pass this code inside the try block